Checkout/Checkout/Source/Validation/Constants.swift

```diff
@@ -7,22 +7,23 @@
 
 import Foundation
 
-enum Constants {
-  enum Address {
-    static let addressLine1Length = 200
-    static let addressLine2Length = 200
-    static let cityLength = 50
-    static let stateLength = 50
-    static let zipLength = 50
-    static let countryLength = 2
-  }
+public enum Constants {
+    public enum Address {
+        public static let addressLine1Length = 200
+        public static let addressLine2Length = 200
+        public static let cityLength = 50
+        public static let stateLength = 50
+        public static let zipLength = 50
+        public static let countryLength = 2
+    }
+
+    public enum Phone {
+        public static let phoneMinLength = 6
+        public static let phoneMaxLength = 25
+        public static let countryCodeMinLength = 1
+        public static let countryCodeMaxLength = 7
+    }
```

Comment:
Do we need to make all of these properties public? Tried looking for some and they don't appear to be used?